ccd-definition/AuthorisationCaseField/CareSupervision/la-barrister.json

```diff
@@ -772,7 +779,7 @@
   {
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "CARE_SUPERVISION_EPO",
-    "CaseFieldID": "manageDocumentsCourtBundleNoHearingsLabel",
```

Comment:
Again think we might need to be careful here removing these

Reply:
see what's been done with the deprecated courtBundleList

Reply:
added the original field to AuthorisationCaseEvent/CareSupervision/DEPRECATED.json

Reply:
reverted. Seems that we don't need to move AuthorisationCaseField to DEPRECATED.json, and there is no such JSON file for AuthorisationCaseField